### ISSUE TITLE
Feature Request: Custom Logo for login page

### ISSUE THREAD

**Issue by rriemann** [netlify/netlify-cms#964]
Hello,

I would like to set a custom logo and a custom text to brand the login page of Netlify CMS.

Best,\
Robert

**Comment by neutyp** [comment]
Good idea; I also think there's room for allowing a small logo to show when in the CMS interface itself. These will need to be addressed separately.

**Comment by erquhart** [comment]
Since each backend provides its own login page, this won't be a core responsibility for the CMS, but we can establish the practice of checking for a logo image path in the config. Agreed on this coinciding with branding the general UI.

**Comment by rriemann** [comment]
The custom text would be the title of our website/organisation. If a div with a dedicated class is used, people can define their own CSS to change the look. if no cusom text as title is set in the config.yml, the div could be omitted.

However, I agree that text could potentially also be added as part of the logo. Especially if svg logos can be used.

**Comment by erquhart** [comment]
I'm okay with using text if no image is provided, that makes sense. Just needs some design direction.

**Comment by mk0r** [comment]
Just commenting to upvote this. In the current login UI the ‘netlifycms’ logo is VERY conspicuous — far bigger than the button for logging in.

Also the text “login with netlify identity” could be confusing for clients who don’t know what Netlify is.

And finally — at the very least “Login” (noun or adjective) should be changed to “Log in” (verb).
